tests/components/lock/test_init.py

```diff
@@ -2,328 +2,295 @@
 from __future__ import annotations
 
 from typing import Any
-from unittest.mock import MagicMock
 
 import pytest
 
 from homeassistant.components.lock import (
     ATTR_CODE,
     CONF_DEFAULT_CODE,
+    DOMAIN,
+    SERVICE_LOCK,
+    SERVICE_OPEN,
+    SERVICE_UNLOCK,
     STATE_JAMMED,
     STATE_LOCKED,
     STATE_LOCKING,
     STATE_UNLOCKED,
     STATE_UNLOCKING,
-    LockEntity,
     LockEntityFeature,
 )
 from homeassistant.core import HomeAssistant
 import homeassistant.helpers.entity_registry as er
-from homeassistant.setup import async_setup_component
-
-from tests.testing_config.custom_components.test.lock import MockLock
-
-
-class MockLockEntity(LockEntity):
-    """Mock lock to use in tests."""
-
-    def __init__(
-        self,
-        code_format: str | None = None,
-        lock_option_default_code: str = "",
-        supported_features: LockEntityFeature = LockEntityFeature(0),
-    ) -> None:
-        """Initialize mock lock entity."""
-        self._attr_supported_features = supported_features
-        self.calls_lock = MagicMock()
-        self.calls_unlock = MagicMock()
-        self.calls_open = MagicMock()
-        if code_format is not None:
-            self._attr_code_format = code_format
-        self._lock_option_default_code = lock_option_default_code
-
-    async def async_lock(self, **kwargs: Any) -> None:
-        """Lock the lock."""
-        self.calls_lock(kwargs)
-        self._attr_is_locking = False
-        self._attr_is_locked = True
-
-    async def async_unlock(self, **kwargs: Any) -> None:
-        """Unlock the lock."""
-        self.calls_unlock(kwargs)
-        self._attr_is_unlocking = False
-        self._attr_is_locked = False
-
-    async def async_open(self, **kwargs: Any) -> None:
-        """Open the door latch."""
-        self.calls_open(kwargs)
-
-
-async def test_lock_default(hass: HomeAssistant) -> None:
-    """Test lock entity with defaults."""
-    lock = MockLockEntity()
-    lock.hass = hass
+from homeassistant.helpers.typing import UNDEFINED, UndefinedType
 
-    assert lock.code_format is None
-    assert lock.state is None
+from .conftest import MockLock
 
+TEST_LOCK_ENTITY_ID = "lock.test_lock"
```

Comment:
Why is this needed when the `mock_lock_entity` fixture returns an entity object?

Reply:
removed it